specification/iotcentral/resource-manager/readme.md

```diff
@@ -0,0 +1,141 @@
+# IotCentral
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for IotCentral.
+
+---
+## Getting Started
+To build the SDK for IotCentral, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information
+These are the global settings for IotCentral.
+
+``` yaml
+openapi-type: arm
+tag: package-2017-07-01-privatepreview
```

Comment:
Is the tag suffix supposed to be "privatepreview"?

Reply:
yes, this is the string we used in the ARM manifest so I kept it the same here to be consistent.